### PR TITLE
Add overlay for Moto E5 Plus

### PR DESCRIPTION
**Pull request by GodcraftPlata**
My previous pull requests were taking values from LineageOS ROM, but since I decompiled stock framework, I used it this time.

### CODE REVIEW

**Comment by TadiT7** [comment]
Delete everything from res/xml/ other than power profile.

**Comment by GodcraftPlata** [comment]
Now everything is perfect :b

File: Moto/E5Plus/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.moto.e5plus"
+        android:versionCode="1"
+        android:versionName="1.0">
+        <overlay android:targetPackage="android"
+                android:requiredSystemPropertyName="ro.vendor.build.fingerprint"
+                android:requiredSystemPropertyValue="motorola/hannah_t/hannah:8.0.0/OCP27.91-150-8/2:user/release-keys"
```
Comment (TadiT7) [inline Moto/E5Plus/AndroidManifest.xml]: You're excluding any other vendor from working with this overlay. Follow these devices: https://github.com/phhusson/vendor_hardware_overlay/tree/master/Moto

Comment (GodcraftPlata) [reply Moto/E5Plus/AndroidManifest.xml]: Do you mean, replace fingerprint with product name?

Comment (GodcraftPlata) [reply Moto/E5Plus/AndroidManifest.xml]: Nevermind, edited it

**Pull request merged**